qemu_v8.mk

```diff
@@ -466,6 +479,27 @@ xen-create-image: linux buildroot | $(XEN_TMP)
 run: all
 	$(MAKE) run-only
 
+ifeq ($(WITH_SCMI),y)
+$(SCMI_DTBO): $(SCMI_DTSO)
+	dtc -I dts -O dtb -o $(SCMI_DTBO) $(SCMI_DTSO)
+
+$(SCMI_DTB): $(SCMI_DTBO)
```

Comment:
Missing dependency on QEMU.
```make
qemu: .stamp_qemu

.stamp_qemu: $(QEMU_BUILD)/config-host.mak
	$(MAKE) -C $(QEMU_PATH)
	touch $@

qemu-clean:
	$(MAKE) -C $(QEMU_PATH) distclean
	rm -f .stamp_qemu

[...]

$(SCMI_DTB): $(SCMI_DTBO) .stamp_qemu
[...]
```

Reply:
IMO `run-only` should not exist. It is here only because the dependencies are not handled in an efficient manner in this makefile (i.e., with timestamp files for sub-projects like I suggest). It would be too slow to have `run:` depend on `qemu`, `linux`, etc. because it would run make in all the sub-projects every time. So this `run-only` hack was implemented. I may propose some cleanup when I find time :wink: 

In any case, `$(SCMI_DTB)` requires QEMU, and without a dependency here you will probably have build errors on many-core machines with `make -j$(nproc)`.

Reply:
Beside this I forgot to check the terminals and create them  if needed but I could remove them from the argument for dumping the dtb as well as initrd and kernel image